content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -16,7 +16,7 @@ weight: 20
 
 {{% capture overview %}}
 
-This document describes the current state of `PersistentVolumes` in Kubernetes. Familiarity with [volumes](/docs/concepts/storage/volumes/) is suggested.
+This document describes the current state of *persistent volumes* in Kubernetes. Familiarity with [volumes](/docs/concepts/storage/volumes/) is suggested.
```

Comment:
why is persistent volumes bolded?

Reply:
To highlight the concept of persistent volumes. The previous version used backquotes resulting in rendering with monospace font that gave the impression that there is an API object called "PersistentVolumes". Since that is not the case, I replaced it with highlighted words "persistent volumes". The same reason applies to changes at many other places in this PR so if there is consensus about the right format, I can apply the same format at other places as well. One option is to simply call them "PVs" as we do in some places.

Reply:
It's not a special word. Please remove the formatting. Thank you. :)

Reply:
@mistyhacks what's the guideline for referring to K8s object names in the plural?

Reply:
When should we use the full object name, when can we abbreviate to PVC, etc?

Reply:
Marking up objects is covered here: https://kubernetes.io/docs/contribute/style/style-guide/#use-camel-case-for-api-objects (use camel-case, a bit further down it says not to split it into separate words like "Persistent Volume Claim")

I struggled to find where we explicitly say that we don't use mono text for object kinds, but I couldn't. I did find instances of it in the style guide itself, and I consider that somewhat definitive. For example: https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language

We don't seem to have any stance about pluralizing, but as far as I know it's normal in Kubernetes to say, for instance, Ingresses. @steveperry-53 @Bradamant3  Do you have thoughts or more background?

I did notice that the PVC page **does** use mono text for the object kinds, so I guess you can leave yours in because the rest of the page could be cleaned up as part of a broader automated sweep. I was being more picky on this CL because it's good to ensure style adherence when a big new block of text is introduced.